myuw/test/model/test_link.py

```diff
@@ -123,8 +123,12 @@ def test_popular_multi_labels(self):
                                       is_student=True, is_seattle=True)
         VisitedLinkNew.objects.create(user=user2, url=TEST_URLS[0], label="x",
                                       is_seattle=True)
+        VisitedLinkNew.objects.create(user=user2, url=TEST_URLS[0], label=None,
+                                      is_seattle=True)
+        VisitedLinkNew.objects.create(user=user3, url=TEST_URLS[0], label=None,
+                                      is_seattle=True)
```

Comment:
ERROR: test_popular_multi_labels (myuw.test.model.test_link.TestLink)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/data/lusers/fanglin/work/myuw-dev/myuw/test/model/test_link.py", line 138, in test_popular_multi_labels
    popular_links = VisitedLinkNew.get_popular()
  File "/data/lusers/fanglin/work/myuw-dev/myuw/models/__init__.py", line 255, in get_popular
    'labels': sorted(by_url[url]['labels'])})
TypeError: '<' not supported between instances of 'str' and 'NoneType'